freex/src/components/Layout/Layout.module.css

```diff
@@ -7,7 +7,6 @@
     "navbar"
     "page_content"
     "footer";
-  margin: 0 60px;
 }
```

Comment:
Myślę że po usunięciu margin z Layout.css który nam to wszystko trzymał w ryzach;) dobrze byłoby zwiększyć max-width na body do 1440px i dodać margin w % albo rem :)